TestApp/update-npm-packages.sh

```diff
@@ -28,10 +28,7 @@ pod repo update
 echo "Install shared framework pods..."
 (cd ../AppCenterReactNativeShared/ios && pod install)
 
-echo "Build shared framework..."
-(cd ../AppCenterReactNativeShared/ios && SRCROOT=`pwd` ./build-fat-framework.sh)
```

Comment:
Remove `build-fat-framework.sh` - it looks. unused now

Reply:
It is used here: https://github.com/microsoft/appcenter-sdk-react-native/blob/develop/AppCenterReactNativeShared/prepare-local-podspec.sh#L4

The main case:
**update-npm-packages.sh** > **pod install**
```
Running pod install...
Auto-linking React Native modules for target `TestApp`: RNCAsyncStorage, RNFS, RNGestureHandler, RNScreens, appcenter-analytics, appcenter-core, appcenter-crashes, and react-native-image-picker
Analyzing dependencies
Fetching podspec for `AppCenterReactNativeShared` from `../../AppCenterReactNativeShared/Products/local.podspec`
```
**prepare-local-podspec.sh** >
**build-fat-framework.sh** >
**zip-framework.sh**